packages/block-editor/src/components/rich-text/index.js

```diff
@@ -39,6 +39,34 @@ import { useFormatTypes } from './use-format-types';
 import FormatEdit from './format-edit';
 import { getMultilineTag, getAllowedFormats } from './utils';
 
+/**
+ * Removes props used for the native version of RichText so that they are not
+ * passed to the DOM element and log warnings.
+ *
+ * @param {Object} props Props to filter.
+ *
+ * @return {Object} Filtered props.
+ */
+function removeNativeProps( props ) {
+	return omit( props, [
+		'__unstableMobileNoFocusOnMount',
+		'deleteEnter',
+		'placeholderTextColor',
+		'textAlign',
+		'selectionColor',
+		'tagsToEliminate',
+		'rootTagsToEliminate',
```

Comment:
What does this do? :)

Reply:
These props are used to clean up any undesired tags produced by Aztec (the native renderer we use for rich text), here is the related code: https://github.com/WordPress/gutenberg/blob/c2905392369e6f423fe43ac17285eb77f4a64d52/packages/rich-text/src/component/index.native.js#L247-L266